### ISSUE TITLE
NTR: Ureidoisobutyric acid high in urine (Lab: urine tests)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8852]
New term request Ureidoisobutyric acid high in urine (elthat_111121051807):

**Comment by MickeySegal** [comment]
In an email I got for https://github.com/obophenotype/human-phenotype-ontology/issues/8853 that seems not to have survived as an entry here you argued from the synonym list https://hmdb.ca/metabolites/HMDB0000026 that N-carbamyl-beta-alanine is the same as https://github.com/obophenotype/human-phenotype-ontology/issues/8852 for Ureidoisobutyric acid, the subject of this NTR. But https://hmdb.ca/metabolites/HMDB0000026 is for Ureidopropionic, not Ureidoisobutyric. So I don't know why this is being closed.

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000622
- label: Elevated urinary ureidoisobutyric acid level
- parents: [Elevated urinary monocarboxylic acid level (HP:0033303)](https://hpo.jax.org/app/browse/term/HP:0033303)
- pmid: PMID:12271438
- def: The amount of ureidoisobutyric acid in the urine, normalized for urine concentration, is above the upper limit of normal. Ureidoisobutyric acid is increased in the urine of patients with beta-ureidopropionase (EC 3.5.1.6) deficiency.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8852
- synonyms:
